Compiled from "AkkaException.scala"
public abstract class akka.OnlyCauseStackTrace$class {
  public static java.lang.Throwable fillInStackTrace(java.lang.Throwable);
    Code:
       0: aload_0
       1: aload_0
       2: invokevirtual #13                 // Method java/lang/Throwable.getCause:()Ljava/lang/Throwable;
       5: astore_1
       6: aload_1
       7: ifnonnull     31
      10: getstatic     #19                 // Field scala/Array$.MODULE$:Lscala/Array$;
      13: getstatic     #24                 // Field scala/reflect/ClassTag$.MODULE$:Lscala/reflect/ClassTag$;
      16: ldc           #26                 // class java/lang/StackTraceElement
      18: invokevirtual #30                 // Method scala/reflect/ClassTag$.apply:(Ljava/lang/Class;)Lscala/reflect/ClassTag;
      21: invokevirtual #34                 // Method scala/Array$.empty:(Lscala/reflect/ClassTag;)Ljava/lang/Object;
      24: checkcast     #36                 // class "[Ljava/lang/StackTraceElement;"
      27: astore_2
      28: goto          36
      31: aload_1
      32: invokevirtual #40                 // Method java/lang/Throwable.getStackTrace:()[Ljava/lang/StackTraceElement;
      35: astore_2
      36: aload_2
      37: invokevirtual #44                 // Method java/lang/Throwable.setStackTrace:([Ljava/lang/StackTraceElement;)V
      40: aload_0
      41: areturn

  public static void $init$(java.lang.Throwable);
    Code:
       0: return
}
